lldb/test/API/tools/lldb-dap/completions/TestDAP_completions.py

```diff
@@ -235,7 +235,7 @@ def test_auto_completions(self):
         """
         Tests completion requests in "repl-mode=auto"
         """
-        self.setup_debugee()
+        self.setup_debugee(stopOnEntry=True)
```

Comment:
Any reason why we are changing the test behavior. I believe currently they will be false.